packages/@dcl/inspector/src/components/tree/Tree.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react'
+import { useDrag, useDrop } from 'react-dnd'
+
+import './Tree.css'
+
+import { Tree, TreeType } from '../../utils/tree'
+import { Input } from './input'
+import { Controls } from './controls'
+import { Icon } from './icon'
+
+interface Props {
+  value: Tree
+}
+
+interface TreeProps {
+  tree: Tree
```

Comment:
The tree here is an instance of a class, where it should be a static type. 

For example:

```ts
type Tree = {
  id: string
  value: string
  label: string
  children: Tree[]
}
```

That's so if we want to make a change to it, we should create a new tree.